apps/enc.c

```diff
@@ -418,15 +426,31 @@ int enc_main(int argc, char **argv)
                     BIO_printf(bio_err, "bad magic number\n");
                     goto end;
```

Comment:
We should really do it at the same time that we are providing new parameters (i.e PBKDF2 and iteration count) so as not to make the problem worse. I'd rather have the discussion now.